### PR TITLE
Fix Cloning App from Gitlab

### PR DESCRIPTION
**Pull request by dvvanessastoiber**

### CODE REVIEW

File: .github/workflows/build-single-product-part.yml [hunk 1]
```diff
@@ -142,7 +142,7 @@ jobs:
           cd "./tmp/$COMPONENT"
           repository="$REPO"
           if [[ $repository == *"gitlab"* && -n $GITLAB_TOKEN ]] ; then
-            repository=${repository/"https://"/"https://$GITLAB_TOKEN@"}
+            repository=https://$GITLAB_TOKEN@${repository}
```
Comment (puehringer) [inline .github/workflows/build-single-product-part.yml:145]: Is this change really necessary if we prefix the repo with `https://`?

Comment (dvvanessastoiber) [reply .github/workflows/build-single-product-part.yml:145]: you are right, thank you! :+1:
